### PR TITLE
Fixed Start-AzAutomationRunbook cmdlet Null reference exception

### PR DESCRIPTION
**Pull request by ikanni**
## Description

Fixed Start-AzAutomationRunbook cmdlet Null reference exception for -Wait option.
Related ICM: Incident 126125279 : CRI - Start-AzAutomationRunbook with -wait causes error: Object reference not set to an instance of an object. | SR:119052423001712

Old PR: https://github.com/Azure/azure-powershell/pull/9503

## Checklist

- [X ] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Automation/Automation/ChangeLog.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 ## Version 1.3.0
 * Fixed Set-AzAutomationConnectionFieldValue cmdlet bug to handle string value.
 * Added text to description for Register-AzAutomationDscNode to explain cross-sub
+* Fixed Start-AzAutomationRunbook cmdlet Null reference exception for -Wait option.
```
[the remark this answers is not in the document]
Comment (ikanni) [reply src/Automation/Automation/ChangeLog.md]: Thank you ! Fixed

**Comment by isra-fel** [comment]
@cormacpayne do we have any updates about mocking GUIDs in playback tests? Thanks!

I have no more problems.

File: src/Automation/Automation.Test/ScenarioTests/AutomationTests.cs [hunk 2]
```diff
@@ -72,9 +72,17 @@ public void TestAutomationStartUnpublishedRunbook()
             TestRunner.RunTestScript("Test-AutomationStartUnpublishedRunbook -runbookPath ScenarioTests\\Resources\\Test-WorkFlowWithVariousParameters.ps1");
         }
 
-        [Fact(Skip = "Need x64 test framework.")]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        // Playback error:
```
[the remark this answers is not in the document]
Comment (ikanni) [reply src/Automation/Automation.Test/ScenarioTests/AutomationTests.cs]: @markcowl, I have removed the test for now. I will schedule a meeting this week

**Comment by ikanni** [comment]
Thank you all for merging this change !

**Pull request merged**